libcxx/include/__math/traits.h

```diff
@@ -34,8 +34,30 @@ namespace __math {
 #  define _LIBCPP_SIGNBIT_CONSTEXPR
 #endif
 
-template <class _A1, __enable_if_t<is_floating_point<_A1>::value, int> = 0>
-_LIBCPP_NODISCARD inline _LIBCPP_SIGNBIT_CONSTEXPR _LIBCPP_HIDE_FROM_ABI bool signbit(_A1 __x) _NOEXCEPT {
+_LIBCPP_NODISCARD inline _LIBCPP_SIGNBIT_CONSTEXPR _LIBCPP_HIDE_FROM_ABI
+#ifdef _LIBCPP_PREFERRED_OVERLOAD
+_LIBCPP_PREFERRED_OVERLOAD
+#endif
```

Comment:
Why can't we instead make this a template like we do for other functions which may have already been provided by the libc?

Reply:
Good idea. I made a proposal. What do you think?

Reply:
I meant that we can instead add `template <class = void>` to make the overloads weaker than the UCRT overloads. Is there any reason not to do that?

Reply:
Now I understand what you meant. Done.